src/main/java/io/ddavison/conductor/util/DriverUtil.java

```diff
@@ -98,12 +103,22 @@ public static WebDriver getDriver(ConductorConfig config) {
         return driver;
     }
 
-    public static void setCustomCapabilities(ConductorConfig config, MutableCapabilities capabilities) {
+    public static MutableCapabilities setCustomCapabilities(ConductorConfig config, MutableCapabilities capabilities, DesiredCapabilities customDesiredCapabilities) {
```

Comment:
We are missing yet _*another*_ look for `MutableCapabilities capabilities`, I think this is way the test is failing. Before this method would just adding the `config` custom caps into the `capabilities` object. Now we are creating a new Capabilities object and we need to add the caps from the config, the original caps, and the new desired caps.